word2gauss/embeddings.pyx

```diff
@@ -1329,7 +1329,7 @@ cpdef np.ndarray[uint32_t, ndim=2, mode='c'] text_to_pairs(
 
     text is a list of text documents / sentences.
 
-    Each element of the list is a numpy array of uint32_t IDs, with -1
+    Each element of the list is a numpy array of uint32_t IDs, with 4294967295
```

Comment:
Because I'm not a C programmer so didn't think of it!  Good tip, I'll change this.